persistent/Database/Persist/Sql/Orphan/PersistQuery.hs

```diff
@@ -335,7 +339,12 @@ filterClauseHelper includeTable includeWhere conn orNull filters =
                                 , qmarks
                                 , ")"
                                 ], notNullVals)
-                            _ -> (name <> showSqlFilter pfilter <> "?" <> orNullSuffix, allVals)
+                            _ -> (T.concat 
+                                [
+                                  if includeExcluded then "EXCLUDED." else ""
```

Comment:
for here, we add the `EXCLUDED` keyword if the `filterClauseHelper` is being called in the context of PostgreSQL.  Is there a more elegant change I can make here?  I don't love how this is an argument that is coming from a boolean that is passed in a call time.